style.css

```diff
@@ -0,0 +1,173 @@
+@media (min-width: 400px) {
+  body {
+    background: #e5e5e5;
+    background-image: url("img/background-mobile.png");
+    background-size: 100%;
+    background-position: absolute;
+    min-height: fit-content;
+  }
+
+  nav {
+    margin-left: 8px;
+  }
+
+  .headline {
+    margin-left: 12px;
+  }
+
+  .main {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .my-logo {
+    width: 76px;
+    height: 20px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+
+    /* identical to box height, or 111% */
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+
+    /* Primary/B400 */
+    color: #6070ff;
+
+    /* Inside auto layout */
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .my-menu div {
+    height: 5px;
+    background-color: #6070ff;
+    margin: 5px 0;
+    border-radius: 25px;
+    transition: 0.5s;
+  }
+
+  .my-menu {
+    width: 30px;
+    display: block;
+  }
+
+  .shape-1 {
+    /* Ellipse 11 */
+    position: absolute;
+    width: 38.5px;
+    height: 19.25px;
+    left: 3.04px;
+    top: 40.47px;
+    border-top-left-radius: 50%;
+    background: #6070ff;
+    transform: rotate(-156.88deg);
+  }
+
+  .shape-2 {
+    /* Rectangle 7 */
+
+    position: absolute;
+    width: 26.46px;
+    height: 4.73px;
+    right: 180.18px;
+    top: 100.59px;
+
+    /* Primary/B400 */
+    background: #6070ff;
+    transform: matrix(0.76, 0.65, 0.65, -0.76, 0, 0);
+  }
+
+  h2 {
+    width: 250px;
+    height: 150px;
+
+    /* Header / 2 */
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+    display: flex;
+    align-items: center;
+
+    /* Neutrals/N800 */
+
+    color: #172b4d;
+
+    /* Inside auto layout */
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .primary {
+    /* Primary Text */
+    width: 300px;
+    height: 146px;
+
+    /* Body / 2 */
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+
+    /* or 150% */
+
+    /* Neutrals/N600 */
+
+    color: #344563;
+
+    /* Inside auto layout */
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .connect {
+    /* LET’S CONNECT */
+
+    width: 200px;
+    height: 24px;
+
+    /* Body / 2 - Bolder */
+
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+
+    /* identical to box height, or 150% */
+
+    /* Primary/B300 */
+    color: #7f8cff;
+
+    /* Inside auto layout */
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  a:hover {
+    color: #6070ff;
+  }
+
+  a:active {
+    color: #2230d2;
+  }
+
+  .socials {
+    width: 100px;
+    display: flex;
+    justify-content: space-around;
+  }
+}
```

Comment:
- [ ] You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). The Minimum mobile screen size should be 375px not 400px.
Your project at 375px Look likes this : 
![cap1](https://user-images.githubusercontent.com/93598189/192008209-066aa372-5823-4b98-9f10-c1ab74ec6cc9.PNG)

- [ ] Even at the Mobile screen size at 400px your Toolbar and Headline section  has  no the proper background image. Your both section should have the margins (top, bottom, left, right) defined in the design .
Your Project at 400px Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/192009166-99bb4b02-500f-4a3d-9de4-c8213df8ac4d.PNG)

The Figma design at the Minimum mobile screen size ( 375px)  Look Likes this : 
  
![Capture1](https://user-images.githubusercontent.com/93598189/192009278-8231ee80-d7e1-4912-b5b3-be6c91bc10e9.PNG)

- [ ] Kindly, I recommend you to work on styling the toolbar and headline section just like the Figma design Template.